redbox-core/redbox/models/chain.py

```diff
@@ -78,6 +86,10 @@ class AISettings(BaseModel):
     # this is also the azure_openai_model
     chat_backend: ChatLLMBackend = ChatLLMBackend()
 
+    # settings for tool call
+    num_retrieve_results: int = 100
```

Comment:
could we give these more descriptive names? 

tool_govuk_retrieved_results
tool_govuk_returned_results

Reply:
updated